scripts/delete-federation.sh

```diff
@@ -34,13 +45,20 @@ function delete-script-deployment() {
 
   # Disable federation of all types
   for ftc in $(kubectl get federatedtypeconfig -n "${NS}" -o=jsonpath={.items..metadata.name}); do
-    ./bin/kubefed2 disable "${ftc}" --delete-from-api --federation-namespace="${NS}"
+    ./bin/kubefed2 disable "${ftc}" --delete-from-api --federation-namespace="${NS}" "${DISABLE_OPTIONS}"
   done
 
   # Remove federation CRDs, namespace, RBAC and deployment resources.
   if [[ ! "${USE_LATEST}" ]]; then
     if [[ "${NAMESPACED}" ]]; then
-      ${KCD} -n "${NS}" -f hack/install-namespaced.yaml
+      if [[ "${DELETE_CLUSTER_RESOURCE}" ]]; then
+        ${KCD} -n "${NS}" -f hack/install-namespaced.yaml
+      else
+        ${KCD} -n "${NS}" role federation-role
```

Comment:
(No action required) This seems like it could become a maintenance hassle.  I guess that's a vote in favor of deprecating script deployment.